### PR TITLE
Feature/andwl 106 manage mnemonic

### PR DESCRIPTION
**Pull request by Kwonhyukjoon**
Android - init mnemonic manage & DB upgrade

### CODE REVIEW

File: app/src/main/java/wannabit/io/cosmostaion/activities/PasswordCheckActivity.java [hunk 1]
```diff
@@ -821,7 +824,11 @@ public void onTaskResponse(TaskResult result) {
 
         } else if (result.taskType == TASK_DELETE_USER) {
             if (result.isSuccess) {
-                onDeleteAccount(mIdToDelete);
+                if (mIdToDelete != -1) {
+                    onDeleteAccount(getBaseDao().onSelectAccount("" + mIdToDelete));
```
Comment (soaryong-c) [inline app/src/main/java/wannabit/io/cosmostaion/activities/PasswordCheckActivity.java:828]: String.valueOf(mIdToDelete)

File: app/src/main/java/wannabit/io/cosmostaion/activities/setting/WalletDeriveActivity.java [hunk 2]
```diff
@@ -0,0 +1,445 @@
...
+                                });
+                            }
+                            @Override
+                            public void onFailure(Call<ResBnbAccountInfo> call, Throwable t) {
+                            }
+                        });
+                    }).start();
+
+                } else if (derive.baseChain.equals(OKEX_MAIN)) {
+                    new Thread(() -> {
```
Comment (soaryong-c) [inline app/src/main/java/wannabit/io/cosmostaion/activities/setting/WalletDeriveActivity.java:337]: remove thread

File: app/src/main/java/wannabit/io/cosmostaion/activities/setting/WalletDeriveActivity.java [hunk 3]
```diff
@@ -0,0 +1,445 @@
...
+                            runOnUiThread(() -> {
+                                WDp.showCoinDp(WalletDeriveActivity.this, getBaseDao(), derive.coin, holder.accountDenom, holder.accountAvailable, derive.baseChain);
+                            });
+                        }
+                    }, derive.baseChain, derive.dpAddress).executeOnExecutor(AsyncTask.THREAD_POOL_EXECUTOR);
+                }).start();
+
+            } else {
+                if (derive.baseChain.equals(BNB_MAIN)) {
+                    new Thread(() -> {
```
Comment (soaryong-c) [inline app/src/main/java/wannabit/io/cosmostaion/activities/setting/WalletDeriveActivity.java:309]: remove thread

**Review by soaryong-c — CHANGES_REQUESTED** [review]

File: app/src/main/java/wannabit/io/cosmostaion/activities/IntroActivity.java [hunk 4]
```diff
@@ -123,26 +126,42 @@ public void onAnimationEnd(Animation animation) {
...
-            Dialog_ChoiceNet dialog = Dialog_ChoiceNet.newInstance(bundle);
-            dialog.setCancelable(true);
-            getSupportFragmentManager().beginTransaction().add(dialog, "dialog").commitNowAllowingStateLoss();
+            Dialog_AddAccount add = Dialog_AddAccount.newInstance(null);
+            add.setCancelable(true);
+            getSupportFragmentManager().beginTransaction().add(add, "dialog").commitNowAllowingStateLoss();
         }
     }
 
+    @SuppressLint("NewApi")
```
Comment (soaryong-c) [inline app/src/main/java/wannabit/io/cosmostaion/activities/IntroActivity.java:140]: remove newapi, update minsdk 24

**Review by soaryong-c — CHANGES_REQUESTED** [review]

**Review by wannabit-yongjoo — APPROVED** [review]

**Pull request merged**